tests/driver_sx127x/main.c

```diff
@@ -302,10 +305,10 @@ static void _event_cb(netdev_t *dev, netdev_event_t event)
             case NETDEV_EVENT_RX_COMPLETE:
```

Comment:
Also, one might reuse the default gnrc_netdev [_event_cb](https://github.com/RIOT-OS/RIOT/blob/master/sys/net/gnrc/link_layer/netdev/gnrc_netdev.c#L49) and the gnrc_netdev thread since gnrc_netdev->recv is pointing to sx127x_adpt_recv. We would need to find a way to handle LoRa specific NETDEV events though...